<?xml version="1.0"?>


<!--EXAMPLE

roslaunch armadillo2 armadillo2.launch sim:=true world_name:="`rospack find robotican_common`/worlds/objects_on_table.world"

roslaunch armadillo2_demos point_head.launch 

-->

<launch>
    
    <group ns="pan_tilt_trajectory_controller">
        <node name="point_head_action" pkg="robotican_common" type="point_head_action" output="screen" >
            <rosparam>
                pan_link:  head_pan_link
                tilt_link: head_tilt_link
                success_angle_threshold: 0.01
            </rosparam>
        </node>
    </group>
    
    
    <node name="find_object_node" pkg="robotican_common" type="find_objects_node" output="screen">
        <param name="~object_id" value="1"/>
        <param name="~depth_topic1" value="/kinect2/qhd/points"/>
        <param name="~depth_topic2" value="/softkinetic/depth/points"/>  
        <rosparam file="$(find armadillo2_demos)/config/red_object.yaml" command="load"/>
    </node>
    
    
    <node name="objects_handler" pkg="robotican_common" type="objects_handler" output="screen">
        <param name="~db_path" value=""/>
    </node> 
    
    
    <node name="pan_tilt_object_tracking_node" pkg="robotican_common" type="pan_tilt_object_tracking_node" output="screen">
        <param name="~object_name" value="can"/>
    </node> 
    
    <node name="object_view" pkg="rqt_image_view" type="rqt_image_view" >
        <remap from="image" to="find_object_node/result" />
    </node>
    
    
    
    
</launch>
